Rebuild npm-shrinkwrap.json; Bump Bower to latest 1.4.1

myabc opened the pull request:
In trying to update (other) dependencies, I've been hitting `npm WARN unmet dependency` errors.
Updating a couple of key dependencies and rebuilding the shrinkwrap file seems to work.

This may be related to this bug: https://github.com/npm/npm/issues/7268

myabc committed:
Bump Bower to latest 1.4.1

See: https://github.com/bower/bower/releases/tag/v1.4.0

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Bump gulp patch version

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Peg gulp-livingstyleguide at 0.1.4

0.1.5 (re-)introduces a dependency on CoffeeScript:
https://github.com/efacilitation/gulp-livingstyleguide/commit/81f9178f84eeb596e72bb11ccdc91f0b7fe96514

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Rebuild npm-shrinkwrap.json

Attempt to deal with `npm WARN unmet dependency` errors.

Signed-off-by: Alex Coles <alex@alexbcoles.com>

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://myabc-fix-npm-rebuild-shrinkwrap.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-fix-npm-rebuild-shrinkwrap.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

myabc committed:
Add workaround for angular-busy template loading

Newer version of ngtemplate-loader are "cleverer" when it comes to
handling the relativeTo param:

    The path for file doesn't contains relativeTo param

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Mark broken inplaceEditor directive tests pending

Pending major refactoring.

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Bump required Webpack version

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc commented:
Closed in favour of reviewing #2763
